trickler/scales.py

```diff
@@ -344,11 +344,91 @@ def _stable_unstable(self, line):
         self.resolution = self.resolution_map[self.unit]
         # Update memcache values.
         self._update_memcache()
+        
+        
+class USSolidScale(SerialScale):
+    """Class for controlling a U.S. Solid USS-DBS81-110G scale."""
+
+    def __init__(self, config, port='/dev/ttyUSB0', baudrate=9600, timeout=0.1, **kwargs):
+        """Only overriding this to provide scale specific constructor arguments."""
+        super().__init__(config=config, port=port, baudrate=baudrate, timeout=timeout, **kwargs)
+
+    @classmethod
+    @property
+    def unit_map(cls):
+        """Mapping of self.unit keys to string units of weight as used by the scale."""
+        return {
+            'gn': cls.Units.GRAINS,
+            'g': cls.Units.GRAMS,
+         }
+
+    @classmethod
+    @property
+    def resolution_map(cls):
+        """Map self.units to matching resolutions with decimal.Decimal values."""
+        return {
+            cls.Units.GRAINS: decimal.Decimal('0.01'),
+            cls.Units.GRAMS: decimal.Decimal('0.001'),
+        }
+
+    # Note(eric): There is no documentation on how to do this for this scale.
```

Comment:
This note is true for the Creedmoor scale. Is it true for yours?

Not going to cause any issues to leave this as-is, but if your scale supports changing the unit over the serial port, the command should be updated to reflect that.

Reply:
I think it can only be changed on the scale itself.

Reply:
Ok. It's probably a good idea to change this function to just the following:

```
def change_unit(self):
"""Changes the unit of weight on the scale."""
    logging.info('This scale does not support changing units through RS232')
    self.update()
```

And remove the `# Note:` line above.